specutils/io/default_loaders/jwst_reader.py

```diff
@@ -393,7 +396,7 @@ def _jwst_s3d_loader(filename, **kwargs):
             slit_header = hdu.header
             header = primary_header.copy()
             header.extend(slit_header, strip=True, update=True)
-            meta = {k: v for k,v in header.items()}
+            meta = {k: v for k, v in header.items()}
```

Comment:
Actually we still have the peculiar situation that this, and only one or two other readers, store the header cards directly in `meta`, while all others use `meta['header']`. Maybe we should first decide on a consistent pattern?
Or do you expect something would break downstream if this were changed for JWST spectra, @jdavies-st?

Reply:
We have not expectations currently on the format of `meta`, so feel free to set a standard and implement it across-the-board.